docs/release-notes.md

```diff
@@ -1,9 +1,51 @@
 # Release notes
 
-## Trino Gateway 10 (24 July 2024)
+## Trino Gateway 11 (5 Sep 2024)
+
+[JAR file gateway-ha-11-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/11/gateway-ha-11-jar-with-dependencies.jar),
+Container image `trinodb/trino-gateway:11`
+
+* [:warning: Breaking change:](#breaking) Require Java 22 for build and runtime.
+  ([#441](https://github.com/trinodb/trino-gateway/pull/441))
+* Add support for using an external service with a REST API to process requests
+  and implement any custom routing rules.
+  ([#423](https://github.com/trinodb/trino-gateway/pull/423))
+* Add support to optionally deactivate adding `X-Forwarded-*` HTTP headers to
+  each request with the `addXForwardedHeaders: false` configuration in 
+  `routing`.
+  ([#417](https://github.com/trinodb/trino-gateway/pull/417))
+* Add OpenMetrics endpoint to
+  [enable monitoring](operation.md#monitoring) with Prometheus and compatible
+  systems.
+  ([#429](https://github.com/trinodb/trino-gateway/pull/429))
+* Add support to [deactivate hostname verification for the certificate of the
+  Trino clusters](security.md#cert-trino).
+  ([#436](https://github.com/trinodb/trino-gateway/pull/436))
+* Add option to use additional, custom `/v1/statement`-like paths as
+  Trino client REST API endpoints on the Trino clusters behind Trino Gateway.
+  ([#326](https://github.com/trinodb/trino-gateway/pull/326))
+* Add support for specifying custom labels in `commonLabels` with the Helm
+  chart. ([#448](https://github.com/trinodb/trino-gateway/pull/448))
+* Fix routing for requests to kill query processing.
+  ([#427](https://github.com/trinodb/trino-gateway/issues/427))
+* Fix routing functionality and query history issues caused by lowercase 
+  HTTP headers in HTTP/2 connections.
+  ([#450](https://github.com/trinodb/trino-gateway/issues/450))
+* Fix issue that broke HTTP/2 requests from client tools to Trino clusters.
+  ([#451](https://github.com/trinodb/trino-gateway/issues/451))
+* Prevent display of wrong user name in the dashboard of the user interface.
+  ([#370](https://github.com/trinodb/trino-gateway/issues/370))
+* Ensure OAuth logout requests are forwarded to the correct Trino cluster that
+  initiated the handshake.
+  ([#455](https://github.com/trinodb/trino-gateway/pull/455))
+
+More details and a list of all merged pull requests are [available in the 
```

Comment:
fyi @oneonestar @willmostly @vishalya @Chaho12  .. should we add that for 10 as well? Thats when we introduced the milestone labelling

Reply:
Sure sounds good :) sometimes it is hard to find history for patches so a link to it sounds great!

Reply:
sgtm, agreed